yaml/eventrouter-deployment.yaml

```diff
@@ -29,15 +29,13 @@ spec:
     spec:
       containers:
         - name: kube-eventrouter
-          image: gcr.io/heptio-images/eventrouter:v0.1
+          args: ['/eventrouter', '-v=10', '-logtostderr']
```

Comment:
I don't really want to default to -v10 though..

Reply:
sorry that got in accidentally.